src/components/navBar/index.tsx

```diff
@@ -1,17 +1,68 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
-
+import { LOGIN_URL, USER_PROFILE_URL } from '@/components/constants/url'
+import Link from 'next/link';
 const RDSLogo = '/RDSLogo.png';
+import useAuthenticated, { DEFAULT_AVATAR } from '@/hooks/useAuthenticated';
+
+const NavBar = () => {
+  const GITHUB_LOGO = '/github-white.png';
+  
+  const {userData, isLoggedIn} = useAuthenticated()
+
+  return (
+    <nav className={styles.navBar}>
+      <div>
+        <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
+        <a href="https://welcome.realdevsquad.com/">Welcome</a>
+        <a href="https://www.realdevsquad.com/events.html">Events</a>
+        <a href="https://members.realdevsquad.com/">Members</a>
+        <a href="https://crypto.realdevsquad.com/">Crypto</a>
+        <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
+      </div>
 
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+      <div>
+        {!isLoggedIn && (
+          <Link href={LOGIN_URL}>
+            <a className={styles.btnLogin}>
+              <button type="button" className={styles.btnLoginText}>
+                Sign In With GitHub
+                <img
+                  className={styles.githubLogo}
+                  src={GITHUB_LOGO}
+                  alt="GitHub Icon"
+                  height="15px"
+                  width="15px"
+                />
+              </button>
+            </a>
+          </Link>
+        )}
+        {isLoggedIn && (
+          <div className={styles.userGreet}>
+            <Link href={USER_PROFILE_URL}>
+              <a>
+                <div className={styles.userGreetMsg}>
+                  {isLoggedIn
```

Comment:
can we use a ternary operator here like [this](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator)? Also we can make these two as a separate component and then use it here

Reply:
done